### ISSUE TITLE
Localize Application Programming Interface (API) page in Hindi 

### ISSUE THREAD

**Issue by VamshiReddy02** [cncf/glossary#1624]
The application Programming Interface (API) page is available in English and Bengali. Still, This term also needs to be Localized into Hindi for the people who want to read this term in Hindi.
Link: https://glossary.cncf.io/application-programming-interface/

**Comment by SaxenaAnushka102** [comment]
Hey @VamshiReddy02, this idea sounds cool! Do we need to translate this into Hindi?

**Comment by VamshiReddy02** [comment]
YES! @SaxenaAnushka102  we need to translate this page into Hindi.

**Comment by SaxenaAnushka102** [comment]
I'd like to contibute. Please could you assign?

**Comment by Aarav238** [comment]
I would like to work on it . Please assign this to me

**Comment by SaxenaAnushka102** [comment]
Hey @Aarav238! I've already submitted a PR for this issue which is yet to be reviewed. I think the maintainers are busy. It would be great if you could localize some other different terms in the glossary.

**Comment by Aarav238** [comment]
Okay @SaxenaAnushka102 . No issue.

**Comment by SaxenaAnushka102** [comment]
Hey @Garima-Negi @anubha-v-ardhan!
I would like to localize `application-programming-interface.md` to Hindi. Here's a placeholder PR for the same.
https://github.com/cncf/glossary/pull/1632

**Comment by divyansh2375** [comment]
I would like to work on it
